textfsm_test.py

```diff
@@ -513,15 +513,22 @@ def testParseTextToDicts(self):
     # Tests 'Filldown' and 'Required' options.
     data = 'two\none'
     result = t.ParseTextToDicts(data)
-    self.assertEqual(str(result), "[{'hoo': 'two', 'boo': 'one'}]")
+    try:
```

Comment:
There is another Pull out that will fixe this (v3 support)

Reply:
Thanks, that looks like it has a bunch of useful stuff in. I'll wait for it to get accepted then remove this clutter.